zcnbridge/bridge_test.go

```diff
@@ -39,7 +40,12 @@ import (
 
 const (
 	ethereumAddress = "0xD8c9156e782C68EE671C09b6b92de76C97948432"
-	password        = "02289b9"
+
+	alchemyEthereumNodeURL = "https://eth-mainnet.g.alchemy.com/v2/9VanLUbRE0pLmDHwCHGJlhs9GHosrfD9"
+	infuraEthereumNodeURL  = "https://mainnet.infura.io/v3/7238211010344719ad14a89db874158c"
+	value                  = 1e+10
+
+	password = "02289b9"
```

Comment:
Let's put the info above into github ENVs or SECRETS

Reply:
The given data are fake, there is not security issue.

Reply:
It's used only for tests